p2p/host/basic/basic_host.go

```diff
@@ -536,11 +497,6 @@ func (h *BasicHost) Close() error {
 	return h.proc.Close()
 }
 
-// GetBandwidthReporter exposes the Host's bandiwth metrics reporter
-func (h *BasicHost) GetBandwidthReporter() metrics.Reporter {
```

Comment:
This isn't a part of the Host interface and I can't find anything that actually uses it. Really, users should just hang onto the bandwidth reporter that they passed in.